src/components/10-atoms/input-file/react/story.jsx

```diff
@@ -1,116 +1,62 @@
 /* global document */
 import { storiesOf } from '@storybook/html';
+import { text, select, boolean, withKnobs } from '@storybook/addon-knobs';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import AXAInputFileReact from './AXAInputFileReact';
 import Readme from '../README.md';
 
+export const iconOptions = {
```

Comment:
Why is this variable exported? That makes no sense imho.

Reply:
I know now why and we should not do this, because it makes the component a "folder".